### PR TITLE
Mockdata and example endpoint

### PR DESCRIPTION
**Pull request by johnolos**

### CODE REVIEW

File: src/main/scala/io/pubcrawler/stout/db/TableDefinitions.scala [hunk 1]
```diff
@@ -26,14 +25,14 @@ trait TableDefinitions {
     def * = (id.?, ownerId) <> (Route.tupled, Route.unapply)
   }
 
-  val routes: TableQuery[Routes] = TableQuery[Routes]
+  val _routes: TableQuery[Routes] = TableQuery[Routes]
```
Comment (johnolos) [inline src/main/scala/io/pubcrawler/stout/db/TableDefinitions.scala]: routes is a thing in scalatra :-(
Open to rename all variables to remain consistent.

File: src/main/scala/io/pubcrawler/stout/util/JsonFormat.scala [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+trait JsonFormat {
+  implicit lazy val formats: Formats = DefaultFormats ++ JavaTimeSerializers.defaults ++ EnumNameSerializers.defaults
+}
+
+object EnumNameSerializers {
+  val defaults = new EnumNameSerializer(Gender) :: new EnumNameSerializer(Status) :: Nil
+}
+
+object JavaTimeSerializers {
+  // Borrowed from https://github.com/meetup/json4s-java-time/../JavaTimeSerializers.scala
```
Comment (johnolos) [inline src/main/scala/io/pubcrawler/stout/util/JsonFormat.scala:22]: :sweat_smile:

File: src/main/scala/io/pubcrawler/stout/db/TableDefinitions.scala [hunk 3]
```diff
@@ -26,14 +25,14 @@ trait TableDefinitions {
...
 
   class Crawls(tag: Tag) extends Table[Crawl](tag, "crawls") {
     def id = column[Int]("crawl_id", O.PrimaryKey, O.AutoInc)
     def title = column[String]("title")
     def ownerId = column[Int]("owner_id")
     def routeId = column[Int]("route_id")
-    def dateTime = column[Time]("date_time")
+    def dateTime = column[LocalDateTime]("date_time")
```
Comment (johnolos) [inline src/main/scala/io/pubcrawler/stout/db/TableDefinitions.scala:35]: We may need to have a discussion whether we want `LocalDateTime` or `ZonedDateTime`.

**Comment by johnolos** [comment]
I'm not very kind in regards to database migrations, but since we're not up and running, I'm just changing when appropriate.

There's a discussion to be had regarding time-management, but I've personally landed on LocalDateTime being a good option. Let the webapp handle differences in time-zone if they so choose to. For now, I suggest having all time data in UTC.

**Review by hessenh — APPROVED** [review]

**Pull request merged**